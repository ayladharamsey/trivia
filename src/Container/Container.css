.card {
    height: 250px;
    width: 200px;
    background-color: #0B0C10;
    border: 3px solid #66FCF1;
    border-radius: 5px;
}
