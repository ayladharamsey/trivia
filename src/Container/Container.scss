section {
  display: grid;
  width: 100%;
  grid-template-areas:
    "nav nav nav nav nav"
    "user user user user user"
    "container container container container container";

  grid-template-columns: repeat(5, 1fr);
  grid-auto-rows: auto;

  h2 {
    color: white;
    margin: auto;
    margin-top: 50px;
  }

  .card {
    display: block;
    box-sizing: border-box;
    margin: 10px;
    height: 220px;
    width: 250px;
    background: rgba(155, 155, 155, .2);
    padding-left: 10px;
    color: white;
    border: 3px solid #66FCF1;
    border-radius: 5px;
  }

}