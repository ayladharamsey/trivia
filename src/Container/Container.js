import React from 'react';
import './Container.css';

const Container = () => {

}

export default Container;