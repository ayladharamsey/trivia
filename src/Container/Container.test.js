import React from 'react';
import { shallow } from 'enzyme';
import Container from './Container';

describe('Container', () => {

  beforeEach(() => {

  });

  it('should do something cool', () => {

  });


});