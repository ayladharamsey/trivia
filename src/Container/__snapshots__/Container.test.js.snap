// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match the snapshot 1`] = `
<section>
  <section
    className="card-components"
  >
    <MovieCard
      characters={
        Array [
          "url.com",
          "url.com",
          "url.com",
        ]
      }
      episode={1}
      id={1}
      release_year="2002-05-16"
      title="starwars"
    />
  </section>
  <section
    className="card-components"
  >
    <MovieCard
      characters={
        Array [
          "url.com",
          "url.com",
          "url.com",
        ]
      }
      episode={1}
      id={1}
      release_year="2002-05-16"
      title="starwars"
    />
  </section>
  <section
    className="card-components"
  >
    <MovieCard
      characters={
        Array [
          "url.com",
          "url.com",
          "url.com",
        ]
      }
      episode={1}
      id={1}
      release_year="2002-05-16"
      title="starwars"
    />
  </section>
</section>
`;
